src/arch/sparc/include/asm/ptrace.h

```diff
@@ -29,6 +29,12 @@ struct pt_regs {
 	uint32_t ins[8];     /**< Input registers. */
 }__attribute__ ((aligned (8)));
 
+#define PT_REGS_ALIGN    8
```

Comment:
Are other implementations (that are possible on SPARC) require same alignment, or just this one? For example we could not simulate operating with stack.
IMO, In this particular case, concrete implementation should not make so big impact on high-level structure. Having alignment is useful, for performance reasons for example, but as far as I understand that is not the reason this alignment appeared.